client/src/components/payment/refund-modal.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <v-dialog width="40rem" :value="isVisible" persistent v-if="payments">
+    <v-card>
+
+      <v-card-title class="d-flex justify-space-between">
+        <div>Cancel and Refund</div>
+        <v-btn icon large color="primary" class="dialog-close" @click="hideModal()">
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </v-card-title>
+
+      <v-card-text class="text-body-1">
```

Comment:
Note Vuetify class "text-body-1" (and text-body-2 below).

Reply:
i like it!

Reply:
Yes, instead of using a local/custom style for the larger first para text and the smaller bottom para text, I leveraged the Vuetify styles. (I only had to override the Vuetify font face.)